### PR TITLE
Show the Configure Events page for non org admin users (Stage)

### PR DESCRIPTION
**Pull request by CodyWMitchell**
[RHCLOUD-30431](https://issues.redhat.com/browse/RHCLOUD-30431)
- Allow users with `integrations:*:*` , `integrations:endpoints:write` to view the Configure Events page in stage beta and preview

### CODE REVIEW

File: static/stable/stage/navigation/settings-navigation.json [hunk 1]
```diff
@@ -54,7 +54,12 @@
                   "href": "/settings/notifications/configure-events",
                   "permissions": [
                     {
-                      "method": "isOrgAdmin"
+                        "method": "loosePermissions",
+                        "args": [
+                            "isOrgAdmin",
```
Comment (karelhala) [inline static/stable/stage/navigation/settings-navigation.json:59]: Same as above.
```suggestion
```

File: static/beta/stage/navigation/settings-navigation.json [hunk 2]
```diff
@@ -68,7 +68,12 @@
                     "href": "/settings/notifications/configure-events",
                     "permissions": [
                       {
-                        "method": "isOrgAdmin"
+                        "method": "loosePermissions",
+                          "args": [
+                            "isOrgAdmin",
```
Comment (karelhala) [inline static/beta/stage/navigation/settings-navigation.json:73]: This won't work, the `isOrgAdmin` should be another permission method. @Hyperkid123 do we have `or` between permission methods or `and`?
```suggestion
```

Comment (Hyperkid123) [reply static/beta/stage/navigation/settings-navigation.json:73]: We only have `and`

The permission `isOrgAdmin` does not exist. However, org admin should have the specific permission anyway. So finding using permission strings should work. I might be wrong.

**Review by karelhala — APPROVED** [review]
Let's try this approach, org admins should have either integrations all or write permissions anyway.

**Pull request merged**